CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Changelog
 Unreleased
 ----------
 
+### Added
+* Added an environment variable to support WebdriverIO's bail option.
```

Comment:
Lets add the name of the env var

Reply:
Done: https://github.com/cerner/terra-toolkit/pull/269/commits/7b3ff34158bbe8a01c4dc893a823947e5dad2e66.